src/Symfony/Component/HttpKernel/Event/GetResponseForExceptionEvent.php

```diff
@@ -45,7 +41,19 @@ public function __construct(HttpKernelInterface $kernel, Request $request, int $
      */
     public function getException()
     {
-        return $this->exception;
+        if (!$this->throwable instanceof \Exception) {
+            return new ErrorException($this->throwable);
+        }
+
+        return $this->throwable;
+    }
+
+    /**
+     * @internal
+     */
+    public function getThrowable(): \Throwable
```

Comment:
@Tobion We add this method on 4.4 so we can get throwables everywhere in our code base. On master, we remove it and use `getException()` everywhere (we obviously remove the conversion done in `getException()`.

Reply:
seems fine. please add the information to the `@internal` annotation so people know why it's internal